### PR TITLE
Changes needed by appveyor build cucumber-cpp

### PR DESCRIPTION
**Pull request by konserw**
These are changes needed by my appveyor build branch of cucumber-cpp. (as I couldn't figure out how to make "run_simple" to work on windows)
Please merge to cucumber-cpp branch.

### CODE REVIEW

**Comment by konserw** [comment]
Please?

File: step_definitions/cucumber_mappings.rb [hunk 1]
```diff
@@ -50,7 +50,7 @@ def step_file(pattern)
   end
 
   def pattern_exists?(pattern)
-    File.exist?(File.join(current_dir, step_file(pattern)))
+    File.exist?(File.join(expand_path("."), step_file(pattern)))
```
Comment (paoloambrosio) [inline step_definitions/cucumber_mappings.rb:53]: Done the same thing in 11d6379b9096f6aa9920e2861bd0530ca7c6302b. Didn't see this was a separate commit. Sorry, you'll have to rebase skipping this commit :-/

**Comment by paoloambrosio** [comment]
What are the other changes for?

File: step_definitions/cucumber_mappings.rb (hunk 1 above)
Comment (konserw) [inline step_definitions/cucumber_mappings.rb:53]: If you wanted merging this separatelly why haven't you just marge #18 ?

Comment (paoloambrosio) [inline step_definitions/cucumber_mappings.rb:53]: After merging and rebasing a few PRs I lost track of what was where. Everyone makes mistakes ;-)

Comment (konserw) [inline step_definitions/cucumber_mappings.rb:53]: Fair enough :)

**Comment by konserw** [comment]
It is due to run_simple from arub doesn't work on windows, so I changed it to:

```
 -    run_simple "cucumber #{params} #{FEATURES_DIR}", false
+    cmd="cucumber #{params} #{FEATURES_DIR}"
+    @output = %x[ #{cmd} ]
+    @return = ($?.success?)
```

And then - `assert_success` does not work.
to be honest I don't know why those asserations are inside tck and other asserations are to be defined in step mappings, like `assert_passing_scenario`

**Comment by paoloambrosio** [comment]
Not needed. See cucumber/cucumber-cpp#113 and comments on cucumber/cucumber-cpp#97.

**Pull request closed without merging**